cmd/devdata/main.go

```diff
@@ -182,10 +182,15 @@ func makeSystemIntake(name string, logger *zap.Logger, store *storage.Store, cal
 
 		ProjectName:     null.StringFrom(name),
 		ExistingFunding: null.BoolFrom(true),
-		FundingNumber:   null.StringFrom("123456"),
-		FundingSource:   null.StringFrom("Research"),
-		BusinessNeed:    null.StringFrom("A business need. TACO is a new tool for customers to access consolidated Active health information and facilitate the new Medicare process. The purpose is to provide a more integrated and unified customer service experience."),
-		Solution:        null.StringFrom("A solution. TACO is a new tool for customers to access consolidated Active health information and facilitate the new Medicare process. The purpose is to provide a more integrated and unified customer service experience."),
+		FundingSources: []*models.SystemIntakeFundingSource{
```

Comment:
This doesn't quite work to seed the funding sources, because `UpdateSystemIntakeFundingSources()` doesn't get called. (I'm not sure if we want to change `CreateSystemIntake()`/`UpdateSystemIntake()` to also update funding sources).

If we don't modify `CreateSystemIntake()`/`UpdateSystemIntake()`, what you can do to seed funding sources data is initialize the funding sources after creating/updating the intakes, then call `UpdateSystemIntakeFundingSources()`. (Make sure to pass `intake.ID` to the funding sources) Something like this:

```go
fundingSources := []*models.SystemIntakeFundingSource{
		{
			SystemIntakeID: intake.ID,
			FundingNumber:  null.StringFrom("123456"),
			Source:         null.StringFrom("Research"),
		},
		{
			SystemIntakeID: intake.ID,
			FundingNumber:  null.StringFrom("789012"),
			Source:         null.StringFrom("DARPA"),
		},
}

must(store.UpdateSystemIntakeFundingSources(ctx, intake.ID, fundingSources))
```

Reply:
Ah good catch